lib/recurly/resource.rb

```diff
@@ -437,7 +437,9 @@ def from_xml xml
           else
             val = XML.cast(el)
             if 'address' == el.name && val.kind_of?(Hash)
-              record[el.name] = Address.new val
+              address = Address.new val
+              address.instance_variable_set(:@changed_attributes, {})
```

Comment:
When the address is unmarshalled, it's fields are dirty. Not sure why. This is too hacky.